docs/user-guide/monitoring.md

```diff
@@ -0,0 +1,161 @@
+# Prometheus and Grafana installation
+
+This tutorial will show you how to install [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) for the metric scraping from the NGINX Ingress controller.
+
+## Before You Begin
+
+Now, deploy the Ingress controller following the instructions [here](../deploy/index.md).
```

Comment:
Since it's a prerequisite I'd change the imperative form "Now, ..." to "The NGINX Ingress controller should already be deployed according to the deployment instructions. See [Deployment] (...)"